lib/playgroundbook_renderer/page_writer.rb

```diff
@@ -22,14 +22,14 @@ def write_page(page_name, page_dir_name, imports, page_contents)
         end
 
         File.open(MANIFEST_FILE_NAME, 'w') do |file|
-          file.write ({
+          file.write {
             'Name' => page_name,
             'LiveViewMode' => 'HiddenByDefault',
             'Version' => '1.0',
-            'ContentVersion' => '1.0',
-          }.to_plist)
+            'ContentVersion' => '1.0'
+          }.to_plist
```

Comment:
Yeah, I fixed this on #29 - the autocorrect actually changes the semantics of the return value